aqua-examples/near-integration/README.md

```diff
@@ -89,11 +128,13 @@ Which produces output similar to:
 
 ```bash
 > near-signing-node@0.1.0 compile-aqua
-> aqua -i aqua/ -o src/_aqua
+> fluence aqua -i aqua/ -o src/_aqua
```

Comment:
should use fluence cli to set up the project and including js part of the project starting with fluence init.

Reply:
For the JS part it's not necessary to focus on init due to the following:

- the fluence cli doesn't guide that JS part of a JS developer workflow but rather extends it a bit (in our case just 1) wraps the aqua cli call and 2) making aqua functions run more convenient)
- considering the above, I've put [fluence.yaml](https://github.com/fluencelabs/examples/blob/near-fluence-cli/aqua-examples/near-integration/near-signing-node/fluence.yaml) in the js part the fluence cli doesn't ask you to do init in the directory.

So, instead of init for a new project, you have a project in a repo already.

Reply:
k. thx.